pyproject.toml

```diff
@@ -1,8 +1,4 @@
 [tool.black]
 line-length = 88
 target-version = ['py36', 'py37', 'py38']
-exclude = '''
-/(
-  scripts
-)/
-'''
+exclude = '/\{\{ cookiecutter.repo_directory \}\}/'
```

Comment:
What is this change about?

Reply:
Just because some Python files are now within the tf modules templates, and they should be excluded from Black formatting. I don't think 'scripts' was needed here as an exclusion.